GliaWidgets/Sources/CallVisualizer/Coordinator/CallVisualizer.Coordinator.swift

```diff
@@ -131,6 +131,9 @@ extension CallVisualizer {
             let alert = AlertViewController(
                 kind: .liveObservationConfirmation(
                     environment.viewFactory.theme.alertConfiguration.liveObservationConfirmation,
+                    link: { _ in
+                        // TODO: Add navigating to WebView controller
```

Comment:
Do you plan to implement it in separate ticket or this has to be done in current PR?

Reply:
I'm going to add this in separate PR